.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: cfn-module-generate
+    name: CloudFormation Module generate
+    entry: cfn generate
+    language: python
+    files: ^fragments\/.*\.(json|yaml|yml)$
```

Comment:
Is this `files:` directive actually needed in this context?  Would not the `cfn generate` command, running at the project's root, just suffice in this case?

Reply:
thinking was that the hook only needs to run if the fragment is modified, otherwise can be skipped.